### PR TITLE
fix(db-browser): Support search table or view by identityNameLike

### PR DESCRIPTION
**Pull request by ysjemmm**
#### What type of PR is this?
type-bug

### CODE REVIEW

File: libs/db-browser/src/test/java/com/oceanbase/tools/dbbrowser/schema/MySQLNoLessThan5700SchemaAccessorTest.java [hunk 1]
```diff
@@ -192,7 +192,7 @@ public void listSystemViews_information_schema_not_empty() {
 
     @Test
     public void listAllSystemViews_Success() {
-        List<DBObjectIdentity> sysViews = accessor.listAllSystemViews();
+        List<DBObjectIdentity> sysViews = accessor.listAllSystemViews(null);
```
Comment (MarkPotato777) [inline libs/db-browser/src/test/java/com/oceanbase/tools/dbbrowser/schema/MySQLNoLessThan5700SchemaAccessorTest.java:195]: Please add test cases for those names are not null.

**Review by MarkPotato777 — APPROVED** [review]
LGTM

File: server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DBMetadataController.java [hunk 2]
```diff
@@ -50,12 +51,14 @@ public class DBMetadataController {
...
             @RequestParam(required = false, name = "type") List<DBObjectType> types,
+            @RequestParam(required = false, name = "schemaName") String schemaName,
             @RequestParam(required = false, name = "identityNameLike") String identityNameLike) {
         ConnectionSession session = sessionService.nullSafeGet(SidUtils.getSessionId(sessionId), true);
         if (ConnectionSessionUtil.isLogicalSession(session)) {
             return Responses.list(ListUtils.EMPTY_LIST);
         }
-        return Responses.list(identitiesService.list(session, identityNameLike, types));
+        return Responses
+                .list(identitiesService.list(session, schemaName, StringUtils.escapeLike(identityNameLike), types));
```
Comment (MarkPotato777) [inline server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DBMetadataController.java:61]: The escape logic should be put in db-browser, rather than the service side.

**Review by MarkPotato777 — APPROVED** [review]
LGTM

**Review by guowl3 — APPROVED** [review]
